samples/http-json-cloudevents/README.md

```diff
@@ -0,0 +1,20 @@
+## Quickstart
+
+Install dependencies:
+
+```sh
+pip3 install -r requirements.txt
+```
+
+Start server:
+
+```sh
+python3 server.py
+```
+
+In a new shell, run the client code which sends a structured and binary
+cloudevent to your local server:
+
+```sh
+python3 client.py http://localhost:3000/
+```
```

Comment:
How do I run tests?

Reply:
added test h1 tag. Should this be just normal text?